docs/UserGuide.md

```diff
@@ -1,9 +1,8 @@
 ---
-layout: page
-title: User Guide
+User Guide
```

Comment:
Maybe just leave it as it is, as `layout` might be required for other stuff (eg. page styling) on the backend by github pages.